File: Exercism/csharp/lucians-luscious-lasagna/LuciansLusciousLasagna.cs
class Lasagna
{
    public int ExpectedMinutesInOven() => 40;

    public int RemainingMinutesInOven(int elapsed) => ExpectedMinutesInOven() - elapsed;

    public int PreparationTimeInMinutes(int layers) => 2 * layers;

    public int ElapsedTimeInMinutes(int layers, int cookingTime) => cookingTime + PreparationTimeInMinutes(layers);
}

File: Exercism/fsharp/lucians-luscious-lasagna/LuciansLusciousLasagna.fs
module LuciansLusciousLasagna

let expectedMinutesInOven = 40

let remainingMinutesInOven elapsed = expectedMinutesInOven - elapsed

let preparationTimeInMinutes layers = 2 * layers

let elapsedTimeInMinutes layers cookingTime = cookingTime + preparationTimeInMinutes layers